src/server/graphql/mutations/changeTaskTeam.js

```diff
@@ -68,7 +69,8 @@ export default {
     const updates = {
       content: rawContent === nextRawContent ? undefined : JSON.stringify(nextRawContent),
       updatedAt: now,
-      teamId
+      teamId,
+      assigneeId: toTeamMemberId(teamId, task.userId)
```

Comment:
Ah, so this is the fix right here. The assigneeId wasn't being updated with the proper teamId, eh?

Reply:
correct. the right way to do it is in #2150, but that'll take an entire day.